### PR TITLE
Bringing down MicroBuild.Core props and targets separately from build_tools

### PR DESCRIPTION
**Pull request by livarcocc**
Bringing down MicroBuild.Core props and targets separately from build_tools.

@dotnet/dotnet-cli

### CODE REVIEW

**Comment by livarcocc** [comment]
@MattGertz for approval. We had our VSO builds in master broken for a while. This hopefully fix them.

File: tools/Signing.Dependencies/Signing.Dependencies.csproj [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (dsplaisted) [inline tools/Signing.Dependencies/Signing.Dependencies.csproj]: I don't see this file referenced anywhere.  Does everything under tools/ get picked up for restore or does this need to be added somewhere?

Comment (livarcocc) [inline tools/Signing.Dependencies/Signing.Dependencies.csproj]: Yes. There is a step in the Prepare.Targets that restores all projects under tools.

**Comment by eerhardt** [comment]
It's still broken. :(

```
2017-05-11T05:15:23.3111873Z ##[error]build\InitRepo.props(1,1): Error MSB4041: The default XML namespace of the project must be the MSBuild XML namespace. If the project is authored in the MSBuild 2003 format, please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.
2017-05-11T05:15:23.3111873Z E:\A\_work\564\s\build\InitRepo.props(1,1): error MSB4041: The default XML namespace of the project must be the MSBuild XML namespace. If the project is authored in the MSBuild 2003 format, please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format. [E:\A\_work\564\s\build\Signing.proj]
2017-05-11T05:15:23.3111873Z Done Building Project "E:\A\_work\564\s\build\Signing.proj" (PostCompileSign target(s)) -- FAILED.
```

Signing uses VS2015 MSBuild, so you need to ensure anything it imports doesn't use new VS2017 MSBuild features.

**Pull request merged**